manager/allocator/networkallocator/networkallocator.go

```diff
@@ -21,6 +22,13 @@ const (
 	// default if a network without any driver name specified is
 	// created.
 	DefaultDriver = "overlay"
+	// PredefinedNetworkLabel is the swarmkit label attached to a network object
+	// representing one of the predefined networks known to be statically
+	// created on the cluster nodes. These objects are populated in the store
+	// at cluster creation solely in order to support running services on the
+	// nodes' predefined networks. As a consequence, these networks need not
+	// to be exposed via public APIs.
+	PredefinedNetworkLabel = "com.docker.swarm.predefinedNetwork"
```

Comment:
what about `com.docker.swarm.networks.predefined`, or by the fact that we set the label into a network spec then it is enough `com.docker.swarm.predefined` like you said. Will change as you suggested.